src/main/java/Contexts/SearchResultContext.java

```diff
@@ -0,0 +1,98 @@
+package Contexts;
+
+import javafx.util.Pair;
+import object.MovieInfoObject;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+
+public class SearchResultContext {
+
+
+    static ArrayList<String> keywords = new ArrayList<>();
+
+    static ArrayList<MovieInfoObject> mMovies = new ArrayList<>();
+
+    static ArrayList<MovieInfoObject> mCurrentMovies = new ArrayList<>();
+
+    public static void initialiseContext(String[] listOfKeys) {
+        for (String a:listOfKeys) {
```

Comment:
Avoid such variable name `a`.  Provide a meaningful name.

Also, there should be spacing surrounding operators (`keys : listOfKeys`).

Same for those below.